include/sentry.h

```diff
@@ -924,6 +924,16 @@ SENTRY_API int sentry_shutdown(void);
  */
 SENTRY_EXPERIMENTAL_API void sentry_clear_modulecache(void);
 
+/**
+ * Re-initializes the Sentry backend
+ *
+ * This is mainly meant for if a third-party library hijacks Sentry's signal
```

Comment:
I would maybe phrase this more positively:

This is needed if a third-party library overrides the previously installed signal handler, and reinstalls the Sentry handler.